Add basic GH200 hardware description (#118)
eth-cscs/cscs-docs: docs/alps/hardware.md

> There are currently four node types in Alps, with another becoming available in 2025:

Reviewer: ```suggestion
There are currently five node types in Alps:
```

| type           | abbreviation  | blades | nodes | CPU sockets | GPU devices |
| ----           | -------       | ------:| -----:| -----------:| -----------:|
| NVIDIA GH200   | gh200         | 1344   | 2688  | 10,752      | 10,752      |
| AMD Rome       | zen2          |  256   | 1024  |  2,048      | --          |
| NVIDIA A100    | a100          |   72   |  144  |    144      | 576         |

> | AMD MI250x     | mi200         |   12   |   24  |     24      |  96         |

Reviewer 2: Is `mi200` really the name we use for the MI250x or is it a typo?
```suggestion
| AMD MI250x     | mi250         |   12   |   24  |     24      |  96         |
```

Author: These are the names that we use to label uenv, and to tag runners, etc.
I can remove the column, but I think it is a good idea to have the same "tag" used for all services.

| AMD MI300A     | mi300         |   64   |  128  |    512      | 512         |

[](){#ref-alps-gh200-node}
### NVIDIA GH200 GPU Nodes

There are 24 cabinets, in 4 rows with 6 cabinets per row:

* 8 chassis per cabinet
* 7 blades per chassis

>     * a chassis can contain up to 8 blades, however Alps' gh200 chassis are underpopulated so that we can increase the amount of power delivered to each node.

Reviewer: This is interesting information, but it might be better in a note after the list describing the system as it is?

* 2 nodes per blade

> Each node contains four Grace-Hopper modules and four corresponding network interface cards (NICS) per blade, as illustrated below:

Reviewer: ```suggestion
Each node contains four Grace-Hopper modules and four corresponding network interface cards (NICs) per blade, as illustrated below:
```

![](../images/alps/gh200-schematic.svg)

> ??? info "node xnames"
>     There are two boards per blade with one node per board.
>     This is different to the `zen2` CPU-only nodes (used for example in Eiger) that had two nodes per board for a total of four nodes per blade.
>     As such, there are no `n1` nodes in the xname list, e.g.:
>     ```
>     x1100c0s6b0n0
>     x1100c0s6b1n0
>     ```

Reviewer: Worth adding also a not on how to get the xname of a particular node? 
```
cat /etc/cray/xname
```
```suggestion
??? info "Node xname"
    There are two boards per blade with one node per board.
    This is different to the `zen2` CPU-only nodes (used for example in Eiger) that have two nodes per board for a total of four nodes per blade.
    As such, there are no `n1` nodes in the xname list, e.g.:
    ```
    x1100c0s6b0n0
    x1100c0s6b1n0
    ```
```

There are currently four node types in Alps, with another becoming available in 2025:

| type           | abbreviation  | blades | nodes | CPU sockets | GPU devices |
| ----           | -------       | ------:| -----:| -----------:| -----------:|
| NVIDIA GH200   | gh200         | 1344   | 2688  | 10,752      | 10,752      |
| AMD Rome       | zen2          |  256   | 1024  |  2,048      | --          |
| NVIDIA A100    | a100          |   72   |  144  |    144      | 576         |
| AMD MI250x     | mi200         |   12   |   24  |     24      |  96         |
| AMD MI300A     | mi300         |   64   |  128  |    512      | 512         |

[](){#ref-alps-gh200-node}
### NVIDIA GH200 GPU Nodes

!!! under-construction
    The description of the GH200 nodes is incomplete.

>     Let us know if there is missing information.

Reviewer 2: I feel like this is redundant, we already know that there is missing information (and say so in the sentence before).

I guess you're going for something like "Let us know if there's specific information you'd like to see documented first"? I don't like my wording there, but is that what you had in mind with the sentence?

Author: ![image](https://github.com/user-attachments/assets/93a8b3f2-67e0-47d6-b9dd-1b9adba9af2d)

Reviewer 2: That looks great, thanks @bcumming!
